Automattic/wp-calypso
# Activity: transform rewind/download backup button into ellipsis menu

This PR replaces the primary Rewind button and the dropdown option for a subtler ellipsis menu that contains both. The goal is to reduce the repetition of such a powerful element, which was making it lose its semantic effect.

Ref: p9rlnk-wF-p2 #comment-2892

Additionally, this fixes a couple of related things:

- Improves the copy for the restore option: “Rewind” is now ”Rewind to this point”, to aptly manage expectations from the get go.
- Fixes an issue where the text on the confirmation dialogs (rewind/download backup) went outside its boundaries.
- The ellipsis menu is now vertically centered.

### Before

![image](https://user-images.githubusercontent.com/390760/46820823-01d3d280-cd7f-11e8-9c13-c2b194467567.png)

### After

![image](https://user-images.githubusercontent.com/390760/46820835-09937700-cd7f-11e8-9978-805f60ba802a.png)

![image](https://user-images.githubusercontent.com/390760/46820863-1adc8380-cd7f-11e8-8cc0-fd322f6c4087.png)

### How to test

- Fire this PR up.
- Navigate to a site that has Rewind enabled.
- Check if the ellipsis menu behaves as expected.
- Try to click either the Rewind or Download backup options. Does everything work?
